### PR TITLE
Java: add dlp v2beta2 

### PR DESCRIPTION
**Pull request by neozwu**
Add dlp v2beta2

### CODE REVIEW

File: generated/java/grpc-google-cloud-dlp-v2beta2/build.gradle [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'signing'
+
+description = 'GRPC library for grpc-google-cloud-dlp-v2beta2'
+group = "com.google.api.grpc"
+version = "0.1.26"
```
Comment (garrettjonesgoogle) [inline generated/java/grpc-google-cloud-dlp-v2beta2/build.gradle]: This has been updated to 0.1.27 in the googleapis config

**Comment by neozwu** [comment]
PTAL

**Comment by neozwu** [comment]
@garrettjonesgoogle @andreamlin @vam-google PTAL.  It seems openjdk7 has been failing before this change.

**Comment by garrettjonesgoogle** [comment]
It appears that you regenerated using protoc 3.3; you need to use protoc 3.4. However, if you switch locally, the resource plugin will fail unless you also update your resource plugin to a more recent (unreleased) version which fixes the incompatibility problem with 3.4. It may just be easiest to wait until all the changes in flight have landed.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**